doc/manual/manual.bib

```diff
@@ -11000,4 +11000,11 @@ @article{buit12
 year={2012}
 }
 
-
+@article{guer11,
+  author = 	 {{Guermond, Jean-Luc, Richard Pasquetti, and Bojan Popov}},
+  title = 	 {Entropy viscosity method for nonlinear conservation laws},
+  journal = 	 {Journal of Computational Physics},
+  year = 	 {2011},
+  volume = 	 {230.11},
```

Comment:
the volume is actually only 230. 11 is the issue (which is often completely omitted in references).